zebra-state/src/on_disk.rs

```diff
@@ -295,10 +355,66 @@ impl_from! {
     SerializationError,
     std::io::Error,
     sled::Error,
+    broadcast::RecvError,
 }
 
 impl Into<BoxError> for Error {
     fn into(self) -> BoxError {
         BoxError::from(self.0)
     }
 }
+
+/// A map like interface ontop of sled with a future based interface for
+/// retrieving values that aren't available yet.
+#[allow(dead_code)]
+struct SledMap<K, V> {
+    tree: Tree,
+    pending: HashMap<K, broadcast::Sender<V>>,
```

Comment:
When we get the parent block, we can broadcast the UTXOs to all its child blocks.

But we can't broadcast to any of their descendants, because the child blocks may spend the UTXOs.